skyportal/handlers/api/internal/plot.py

```diff
@@ -47,10 +47,7 @@ def get(self, obj_id):
         device = self.get_query_argument("device", None)
         smoothing = self.get_query_argument("smoothing", False)
         if isinstance(smoothing, str):
```

Comment:
remove the isinstance call, and set `smoothing = self.get_query_argument("smoothing", "false")`